api-docs/swagger.yml

```diff
@@ -307,6 +307,8 @@ paths:
           description: Bad Request
         '404':
           description: Not Found
+        '405':
```

Comment:
Is there a way to comment that this only applies to Swarm?

Reply:
We can amend it in description like this 

`Method Not Allowed. Secret update is not allowed in faas-swarm.`